test/extensions_testsuite/dpl_namespace.pass.cpp

```diff
@@ -40,13 +39,12 @@ int main()
     auto res_first = dpl::begin(res_buf);
     auto counting_first = dpl::counting_iterator<T>(0);
     auto zip_first = dpl::make_zip_iterator(counting_first, key_first);
- 
+
     // key_buf = {0,0,...0,1,1,...,1}
     std::for_each(dpl::execution::make_device_policy<class ForEach>(dpl::execution::dpcpp_default),
 		zip_first, zip_first + n,
-        [](auto x){
-            using std::get;
-            get<1>(x) = (2 * get<0>(x)) / n;
+        [](std::tuple<T, T> x){
```

Comment:
I think `#include <tuple>` is missed there.

Reply:
That's true